.github/templates/common.yml.j2

```diff
@@ -82,8 +82,11 @@ concurrency:
           AWS_RETRY_MODE: standard
           AWS_MAX_ATTEMPTS: 5
         run: |
+          retry () {
```

Comment:
@seemethere I remember you once suggested refactoring this retry somehow to remove duplication across our codebase. How could we do that?

Reply:
We could try to just shuffle it into the `~/.profile` but I guess that's also a bit jank. Another option would be to just create a jinja macro for this so if we need to change this function we only have to change it once instead of multiple times

Reply:
Yea, I went with the jinja macro